plugins/wmm_pi/src/WMM_SubLibrary.c

```diff
@@ -1790,21 +1790,21 @@ WMMtype_LegendreFunction *WMM_AllocateLegendreFunctionMemory(int NumTerms)
 	if (!LegendreFunction) {
 		WMM_Error(1);
 		//printf("error allocating in WMM_AllocateLegendreFunctionMemory\n");
-		return FALSE;
+		return NULL;
 					}
 	LegendreFunction->Pcup = (double *) 	malloc	( (NumTerms +1) * sizeof ( double ) );
 	if (LegendreFunction->Pcup == 0)
 	{
 		WMM_Error(1);
 		//printf("error allocating in WMM_AllocateLegendreFunctionMemory\n");
-		return FALSE;
+        return NULL;

```

Comment:
I think another "rule" was to not make changes just for the sake of whitespace style. But any new commits should use spaces and not tabs. If we start changing tabs alone then commits get larger and more difficult to see changes with actual effect.